### DISCUSSION TITLE
Suggestion of a french little update

### DISCUSSION THREAD

**Discussion by methatronc** [mikeberger/borg_calendar#104] [Ideas]
Hello,
First thanks for your calendar, I have been looking for precisely this for some time, many thanks for bringing it to us !
Also, I'd suggest a small add if you want, I noticed the US and Canadian holidays were put by default, [here](https://etalab.github.io/jours-feries-france-data/ics/jours_feries_metropole.ics) are the French ones if you'd like to add them to the next compilation. Also [here](https://www.data.gouv.fr/fr/datasets/jours-feries-en-france/) is the source since the file will expire in 5 years.
Thanks again and have a good day, you certainly made mine better !

**Comment by mikeberger** [comment]
Hi,

I don't package ICS files in the product - not even for the US and Canadian holidays. Users can find ICS files on the internet and import them. I think I used to have a wiki that talked about importing all kinds of ICS files, but that is long gone.

The US and Canadian holidays are a weird exception since they are calculated in the code and were added in 2003, before I knew what ICS was. The only reason that the Canadian holidays were included is that back then I was working closely with a Canadian team in my job, and I needed to know when they'd be out of the office.

**Reply by methatronc** [reply]
It's always interesting to know how the little things were born, thanks for your answer and indeed, it is easily findable so no worry, it was just a thought.
